### PR TITLE
created a live rates page with some logic and styling

### PR DESCRIPTION
**Pull request by haoxle**
Mock buttons have been added to implement logic,
components imported and styles altered to suit the page
basic styling added

https://trello.com/c/IMAeSaKO/70-create-live-rates-page

### CODE REVIEW

**Review by LewisLovett — CHANGES_REQUESTED** [review]
Resolve merge confilct

File: src/components/LiveRate/LiveRate.jsx [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
-            <p className="live-rate__rate">{!isCurrentCurrency && rate}</p>
-            <Button buttonText={isCurrentCurrency ? "Edit" : "Send"} handleClick={handleClick} />
-        </div>
-    )
-}
+const LiveRate = ({
+  name,
+  amount = "1.00",
+  rate = "0.0037",
+  image_url = "../assets/images/credit-card.png",
```
Comment (olirob93) [inline src/components/LiveRate/LiveRate.jsx:8]: this file path won't work please can you import it in and use it instead

`import Flag from "../../assets/images/credit-card.png"`

`image_url = Flag`

**Review by olirob93 — DISMISSED** [review]
fine for approval once changes are made

**Review by LewisLovett — APPROVED** [review]

File: src/components/LiveRate/LiveRate.scss [hunk 2]
```diff
@@ -2,47 +2,47 @@
 @use "../../styles/_typography.scss" as *;
 
 .live-rate {
-    width: 100vw;
+  // width: 100vw;
```
Comment (kirsty-nology) [inline src/components/LiveRate/LiveRate.scss:5]: remove comment

File: src/pages/LiveRates/LiveRates.jsx [hunk 3]
```diff
@@ -1,8 +1,75 @@
...
+          </div>
+          <div>
+            <h3 id="rates-ctner__title--head">Amount</h3>
+          </div>
+          <div>
+            <h3 id="rates-ctner__title--head">Rate</h3>
+          </div>
+          <div></div>
+          <div> </div>
+          <div></div>
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:54]: why are there 3 empty `div`s here?

File: src/pages/LiveRates/LiveRates.jsx [hunk 4]
```diff
@@ -1,8 +1,75 @@
...
+  const addRate = (e) => {
+    if (!selectCurr.includes(e.target.value)) {
+      setSelectCurr([...selectCurr, e.target.value]);
+    }
+    return;
+  };
+
+  return (
+    <div className="rates">
+      <div className="rates-info">
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:31]: `className=rates__info`

File: src/pages/LiveRates/LiveRates.jsx [hunk 5]
```diff
@@ -1,8 +1,75 @@
...
+        <h1 className="rates-info__transfer"> Transfer </h1>
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:32]: `className=rates__transfer`

File: src/pages/LiveRates/LiveRates.jsx [hunk 6]
```diff
@@ -1,8 +1,75 @@
...
+        <h1 className="rates-info__heading"> View Latest Rates </h1>
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:33]: `className=rates__heading`

File: src/pages/LiveRates/LiveRates.jsx [hunk 7]
```diff
@@ -1,8 +1,75 @@
...
+        <p className="rates-info__content">
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:34]: `className=rates__content`

File: src/pages/LiveRates/LiveRates.jsx [hunk 8]
```diff
@@ -1,8 +1,75 @@
...
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim nam
+          inventore deserunt id? Rem minima dolorem asperiores perferendis sed
+          aspernatur. Dolorem nostrum voluptatibus dolore odit provident minima
+          reiciendis dignissimos iste.
+        </p>
+      </div>
+      <div className="rates-ctner">
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:41]: `className=rates__container`

File: src/pages/LiveRates/LiveRates.jsx [hunk 9]
```diff
@@ -1,8 +1,75 @@
...
+        <div className="rates-ctner__title">
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:42]: `className=rates__title`

File: src/pages/LiveRates/LiveRates.jsx [hunk 10]
```diff
@@ -1,8 +1,75 @@
...
+          <div>
+            <h3 id="rates-ctner__title--head">Currency</h3>
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:44]: `className=rates__title--head`

**Review by AnnaKurochkina — CHANGES_REQUESTED** [review]
Add LoggedIn component to the page please.

File: src/pages/LiveRates/LiveRates.jsx [hunk 11]
```diff
@@ -1,8 +1,75 @@
...
+    () => {
+      const getData = async () => {
+        const currencies = await getCurrencyGBP();
+        setCurrRates(currencies);
+      };
+      getData();
+    },
+    [],
+    selectCurr
+  );
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:20]: curious about the thinking behind this... coming to chat to u in a sec

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

**Review by AnnaKurochkina — DISMISSED** [review]
Can't find a cureency button.
CurrencyButton

**Review by LewisLovett — APPROVED** [review]

File: src/pages/LiveRates/LiveRates.jsx [hunk 12]
```diff
@@ -1,8 +1,87 @@
...
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Enim nam
+          inventore deserunt id? Rem minima dolorem asperiores perferendis sed
+          aspernatur. Dolorem nostrum voluptatibus dolore odit provident minima
+          reiciendis dignissimos iste.
+        </p>
+      </div>
+      <div className="container">
+        <div className="container__title">
+          <div>
+            <h3 id="container__title--head">Currency</h3>
```
Comment (kirsty-nology) [inline src/pages/LiveRates/LiveRates.jsx:63]: `className=container__header`

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

**Review by kirsty-nology — APPROVED** [review]

**Review by LewisLovett — APPROVED** [review]

**Pull request merged**